router/custom-image/acme_router.yaml

```diff
@@ -3,6 +3,7 @@ server:
   cors:
     origins:
       - https://studio.apollographql.com
+  experimental_defer_support: true
```

Comment:
just adding a comment for reference, to test this with a local instance of a graphql studio, on explorer ,you need to add this 

```yml
    allow_any_origin: true # added to allow for local development testing
```